monai/transforms/utility/dictionary.py

```diff
@@ -552,28 +553,21 @@ def __init__(
             device: for Tensor data type, specify the target device.
             wrap_sequence: if `False`, then lists will recursively call this function, default to `True`.
                 E.g., if `False`, `[1, 2]` -> `[tensor(1), tensor(2)]`, if `True`, then `[1, 2]` -> `tensor([1, 2])`.
+            track_meta: whether to convert to `MetaTensor` when `data_type` is "tensor".
+                If False, the output data type will be `torch.Tensor`. Default to the return value of `get_track_meta`.
             allow_missing_keys: don't raise exception if key is missing.
         """
         super().__init__(keys, allow_missing_keys)
-        self.converter = EnsureType(data_type=data_type, dtype=dtype, device=device, wrap_sequence=wrap_sequence)
+        self.converter = EnsureType(
+            data_type=data_type, dtype=dtype, device=device, wrap_sequence=wrap_sequence, track_meta=track_meta
+        )
 
     def __call__(self, data: Mapping[Hashable, NdarrayOrTensor]) -> Dict[Hashable, NdarrayOrTensor]:
         d = dict(data)
         for key in self.key_iterator(d):
-            self.push_transform(d, key)
             d[key] = self.converter(d[key])
         return d
 
-    def inverse(self, data: Mapping[Hashable, Any]) -> Dict[Hashable, Any]:
-        d = deepcopy(dict(data))
-        for key in self.key_iterator(d):
-            # FIXME: currently, only convert tensor data to numpy array or scalar number,
-            # need to also invert numpy array but it's not easy to determine the previous data type
-            d[key] = convert_to_numpy(d[key])
-            # Remove the applied transform
-            self.pop_transform(d, key)
-        return d
-
```

Comment:
Hi @Nic-Ma,
Sorry maybe I didn't explain it well. In this case, inverse() can not use anymore because it has been deleted. Maybe someone had used it may get an error？
Thanks in advance.

Reply:
thanks, I put an inverse with the copy of data. I was not sure about it because of the `wrap_sequence` recursive cases.

Reply:
Hi @KumoLiu ,

I see your point. The `inverse()` operation of `EnsureType` is to convert a Tensor back to numpy array, because we usually use numpy array for transforms.
But now from this version we suppose to use `MetaTensor` for transforms, so I think we don't need to support the inverse support anymore.
And people usually call the inverse operation for the transform chain, deleting the `inverse()` of `EnsureType` will not break the inverse of the chain.

Thanks.